shuup/admin/modules/media/static_src/media/browser/BrowserView.js

```diff
@@ -100,7 +100,18 @@ export function controller(config={}) {
     };
     ctrl.reloadFolderTree = function() {
         remote.get({"action": "folders"}).then(function(response) {
-            ctrl.rootFolder(response.rootFolder);
+            if (response.rootFolder.name === "Root" && !response.rootFolder.canSeeRoot) {
```

Comment:
Idk, maybe checking for `response.rootFolder.id === 0 && ...` would be cleaner instead.